### PR TITLE
feat: Revamp README.md, add badges and better description

### PR DESCRIPTION
**Pull request by AbooMinister25**
Mostly what the title said, I added badges as well as a better description.

### CODE REVIEW

**Review by Objectivitix — APPROVED** [review]
nice html+md action, love it, approved

**Comment by dawnofmidnight** [comment]
Maybe add badges for License and Discord?

File: README.md [hunk 1]
```diff
@@ -1,3 +1,15 @@
-# Catcord
+<h1 align='center'>Catcord</h1>
 
-Catcord is an API created by the cat dev team which holds the backend of a chat application. This API is meant to be requested to in order to get and put data so you can build clients around it to interface with the chat servers. Its built with python and FastAPI and uses PostgreSQL as its database. Anyone can write a client for the API.
+<h2 align="center">A chat application built on a set of API endpoints</h2>
+
+<p align="center">
+    <img src="https://img.shields.io/github/workflow/status/cat-dev-group/catcord/Lint%20Test">
```
Comment (dawnofmidnight) [inline README.md:6]:
```suggestion
    ![Backend Lint & Test](https://img.shields.io/github/workflow/status/cat-dev-group/catcord/Lint%20Test)
```
Try to use markdown images

File: README.md [hunk 2]
```diff
@@ -1,3 +1,15 @@
-# Catcord
+<h1 align='center'>Catcord</h1>
 
-Catcord is an API created by the cat dev team which holds the backend of a chat application. This API is meant to be requested to in order to get and put data so you can build clients around it to interface with the chat servers. Its built with python and FastAPI and uses PostgreSQL as its database. Anyone can write a client for the API.
+<h2 align="center">A chat application built on a set of API endpoints</h2>
+
+<p align="center">
+    <img src="https://img.shields.io/github/workflow/status/cat-dev-group/catcord/Lint%20Test">
+    <img src="https://github.com/cat-dev-group/catcord/workflows/CodeQL/badge.svg">
```
Comment (dawnofmidnight) [inline README.md:7]:
```suggestion
    ![CodeQL](https://github.com/cat-dev-group/catcord/workflows/CodeQL/badge.svg)
```

File: README.md [hunk 3]
```diff
@@ -1,3 +1,15 @@
-# Catcord
+<h1 align='center'>Catcord</h1>
 
-Catcord is an API created by the cat dev team which holds the backend of a chat application. This API is meant to be requested to in order to get and put data so you can build clients around it to interface with the chat servers. Its built with python and FastAPI and uses PostgreSQL as its database. Anyone can write a client for the API.
+<h2 align="center">A chat application built on a set of API endpoints</h2>
+
+<p align="center">
+    <img src="https://img.shields.io/github/workflow/status/cat-dev-group/catcord/Lint%20Test">
+    <img src="https://github.com/cat-dev-group/catcord/workflows/CodeQL/badge.svg">
+    <img src="https://github.com/cat-dev-group/catcord/workflows/Pytest Testing/badge.svg">
```
Comment (dawnofmidnight) [inline README.md:8]:
```suggestion
    ![Pytest](https://github.com/cat-dev-group/catcord/workflows/Pytest%20Testing/badge.svg)

```

**Review by dawnofmidnight — CHANGES_REQUESTED** [review]

**Comment by AbooMinister25** [comment]
Sure, looks good

**Comment by AbooMinister25** [comment]
Why change the `` tags to markdown? I can't center items natively in markdown, and as far as I know, inline HTML is the only way to do that.

**Comment by dawnofmidnight** [comment]
> 
> 
> Why change the `` tags to markdown? I can't center items natively in markdown, and as far as I know, inline HTML is the only way to do that.

You can still center with HTML, but use markdown images, right?

**Comment by AbooMinister25** [comment]
> > Why change the `` tags to markdown? I can't center items natively in markdown, and as far as I know, inline HTML is the only way to do that.
> 
> You can still center with HTML, but use markdown images, right? And since it's mardkown, it's better to keep it as much markdown as you can.

Can you? Huh, thats alright then.

**Comment by AbooMinister25** [comment]
Alright, I've fixed a few things, and fixed the definition. I'll merge this now.

**Pull request merged**